GiEnJul.Test/InfrastructureTests/AutoMapperTest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
These are not in use?

Reply:
yea, forgot to remove after generating class file